hackathon/mdakits/README.md

```diff
@@ -34,7 +34,57 @@ MDAKits paper
   TX, 2023. doi:
   [10.25080/gerudo-f2bc6f59-00a](https://doi.org/10.25080/gerudo-f2bc6f59-00a).
 
+## Getting started
+
+Pick an idea for a stand-alone feature you, or others, want to see 
+possible through MDAnalysis - see the list of Project ideas below!
+We encourage you to discuss any ideas in the lead-up to the hackathon, 
+here or on the UGM discord channel (More details TBA).
+
+Read our guide on making an MDAKit (https://mdakits.mdanalysis.org/makingakit.html).
+(Additional materials TBA.)
+
+Some other things to keep in mind:
```

Comment:
I added this here because these were common things that came up when adding blurbs for the various kit ideas - happy to move if it might fit better elsewhere, though.